softwarereview_author.Rmd

```diff
@@ -4,23 +4,35 @@
--   Do not submit your package for review while it or an associated manuscript is also under review at another venue, as this may result on conflicting requests for changes.
--   Next, [open a new issue](https://github.com/ropensci/software-review/issues/new/choose) in
-the software review repository and fill out the template.
+    -   Do not submit your package for review while it or an associated manuscript is also under review at another venue, as this may result on conflicting requests for changes.
+- If you use [repostatus.org badges](https://www.repostatus.org/) (which we recommend), submit when you're ready to get an _Active_ instead of _WIP_ badge. Similarly, if you use [lifecycle badges](https://www.tidyverse.org/lifecycle/), submission should happen if the package is at least _Maturing_.
+- For any submission or pre-submission inquiry the README of your package should provide enough information about your package (goals, usage, similar packages) for the editors to assess its scope without having to install the package. Even better, set up a pkgdown website for allowing more detailed assessment of functionality online.
+    - At the submission stage, all major functions should be stable enough to be fully documented and tested; the README should make a strong case for the package.
+    - Your README file should strive to explain your package's functionality and aims, assuming readers have little to no domain knowledge. All technical tems, including references to other software, should be clarified.
+- Your package will continue to evolve after review, this book [provides guidance about the topic](#evolution).
+
+## Preparing for Submission
+
+-   Read and follow [our packaging style guide](#building), [reviewer's guide](#preparereview) to ensure your package meets our style and quality criteria.
+-   All submissions are automatically checked by our [pkgcheck](https://docs.ropensci.org/pkgcheck/) system to ensure packages follow our guidelines. All authors are expected to have run [the main `pkgcheck()` function](https://docs.ropensci.org/pkgcheck/reference/pkgcheck.html) locally to confirm that the package may be submitted.
+-   If there are any aspects of `pkgcheck` which your package is unable to pass, please explain reasons in your submission template.
+-   If you feel your package is in scope for the
+    [Journal of Open-Source Software](https://joss.theoj.org/) (JOSS), do not submit it to JOSS consideration until after the rOpenSci review process is over: if your package is deemed in scope by JOSS editors, only the accompanying short paper would be reviewed, (not the software that will have been extended reviewed by rOpenSci by that time). Not all rOpenSci packages will meet the criteria for JOSS.
+
```

Comment:
maybe suggest to ask any question in the rOpenSci forum

Reply:
done - thanks